apps/sms/views/inbox/js/inbox.js

```diff
@@ -55,7 +46,7 @@ var InboxView = {
       'check-uncheck-all-button','composer-link',
       'delete-button', 'edit-header','options-button',
       'settings-button','edit-mode', 'edit-form',
-      'draft-saved-banner'
+      'draft-saved-banner', 'list'
```

Comment:
It's up to you to set a `'click'` delegate on the list and read any attributes you may have set to discern the model. The less fast-list does the better.